src/smtsolvers/Theory.cc

```diff
@@ -106,13 +103,14 @@ CoreSMTSolver::handleSat()
         assert(value(l2) != l_True);
         if (value(l1) == l_Undef && value(l2) == l_Undef) {
             // MB: allocate, attach and remember the clause - treated as original
+            // MB: TODO: why not theory clause?
```

Comment:
Currently the code stores the split clause as an original clause. I guess my question is if this should not be treated as a learnt clause (similar to theory lemmas) rather than an original clause.

Reply:
I see, ok.  That needs some thinking.  We can talk offline, and leave the comment there for now.